runtime/lua/vim/_meta.lua

```diff
@@ -422,6 +422,12 @@ local prepend_value = (function()
     end,
 
     [OptionTypes.ARRAY] = function(left, right)
+      -- prevent prepending before ""
+      -- https://github.com/neovim/neovim/issues/14828
+      if #left == 1 and left[1] == "" then
```

Comment:
Can DRY this check in both functions, but unsure what the function should be named...